### ISSUE TITLE
Python 3 compatibility 

### ISSUE THREAD

**Issue by chhantyal** [euske/pdfminer#63]
Hey @euske, looks like this package doesn't support Python 3 yet. Do you have any plans for it? 

Also let us know if you are looking for help there :)

**Comment by ckolumbus** [comment]
+1

**Comment by SonOfLilit** [comment]
Are you aware of https://pypi.python.org/pypi/pdfminer3k ? I didn't try it, but I did incorporate its workaround of the O(n^2) layout problem.

**Comment by ckolumbus** [comment]
@SonOfLilit I looked at it but didn't try it (yet). Far less activity than pdfminer & last commit is from 2012 and the author's homepage doesn't list it either. That's why I'm hesitant to use it.

**Comment by solstag** [comment]
Is goulu's pdfminer.six <https://github.com/goulu/pdfminer> the 'right place' to go for a python 3 version?

**Comment by zrisher** [comment]
@solstag Yes, it appears to be. See https://github.com/euske/pdfminer/pull/71.

**Comment by zrisher** [comment]
I agree with everything @cdeil said above.
